### PR TITLE
docs(readme.md): api methods table with description

### PR DESCRIPTION
**Pull request by bl0cknumber**
Make ctrl + f over the methods a lot easier on README.

### CODE REVIEW

**Comment by Shinigami92** [comment]
There is a rendering issue

image


---

Beside that, really cool :slightly_smiling_face: 
Do you want to jump into our Discord [Chat on Discord](https://discord.com/invite/4qDjAmDj4P)?

**Comment by fncolon** [comment]
> There is a rendering issue
> 
> image
> 
> Beside that, really cool 🙂 Do you want to jump into our Discord [Chat on Discord](https://discord.com/invite/4qDjAmDj4P)?

Spend 3 hours to test all the faker functions, but it's worth it for me since I always use faker for daily basis.

Yes, i will your discord, my id fndamental#0001.

**Comment by Shinigami92** [comment]
Could you also remove the `See PR #382` stuff?
image

Then I will approve it :raised_hands:

**Comment by fncolon** [comment]
> Could you also remove the `See PR #382` stuff? image
> 
> Then I will approve it 🙌

Sure, i thought it was on current repository, edited.

**Review by Shinigami92 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -61,232 +61,316 @@ var randomCard = faker.helpers.createCard(); // random contact card containing m
...
-  - vehicle
-  - manufacturer
-  - model
-  - type
-  - fuel
-  - vin
-  - color
-  - vrm
-  - bicycle
+Faker provides many useful utilities functions.
```
Comment (import-brain) [inline README.md:64]:
```suggestion
Faker provides many useful utility functions.
```
Other than this minor fix, this PR looks good to me.

**Comment by damienwebdev** [comment]
My opinion is that these should be moved to #82 and then rendered in the https://fakerjs.dev docs. While having in the README is nice, as the API expands we'll see that it becomes cumbersome to maintain. I like the descriptions here, perhaps we can move them into the codebase into the Docblocks if there are any methods missing descriptions?

**Comment by Shinigami92** [comment]
@damienwebdev I exactly thought the same, but for the short run, this is really nice 👍 
We can move it later anyway.

**Comment by fncolon** [comment]
I can help with the docblocks, but merge this to another PR will become larger file changes, maybe if I can suggest I can help for the docblocks in different PR or the PR you mentioned.

**Review by import-brain — DISMISSED** [review]

**Review by Shinigami92 — DISMISSED** [review]

**Review by damienwebdev — APPROVED** [review]
lgtm

**Review by Shinigami92 — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**